### PR TITLE
Set correct event providers for ANCM V2

### PR DESCRIPTION
**Pull request by jkotalik**
For #991.

### CODE REVIEW

File: src/AspNetCoreModuleV2/AspNetCore/src/applicationinfo.cpp [hunk 1]
```diff
@@ -153,7 +153,7 @@ APPLICATION_INFO::UpdateAppOfflineFileHandle()
             STACK_STRU(strEventMsg, 256);
             if (SUCCEEDED(strEventMsg.SafeSnwprintf(
                 ASPNETCORE_EVENT_RECYCLE_APPOFFLINE_MSG,
-                m_pConfiguration->QueryApplicationPath()->QueryStr())))
```
Comment (pakrym) [inline src/AspNetCoreModuleV2/AspNetCore/src/applicationinfo.cpp]: Why change this?

Comment (jkotalik) [reply src/AspNetCoreModuleV2/AspNetCore/src/applicationinfo.cpp]: `m_pConfiguration->QueryApplicationPath()->QueryStr()` is an empty string. It should have been QueryApplicationFullPath. I'm going to investigate what else is using QueryApplicationPath().

Comment (jkotalik) [reply src/AspNetCoreModuleV2/AspNetCore/src/applicationinfo.cpp]: the filePath is the same as the application path except it has app_offline in the path.

Comment (pakrym) [reply src/AspNetCoreModuleV2/AspNetCore/src/applicationinfo.cpp]: Just change it to QueryApplicationPhysicalPath then

**Pull request merged**